.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Ensure that you do not accidentally commit this file into version control by adding it to .gitignore if you intend to keep it private.